sparse.h

```diff
@@ -0,0 +1,48 @@
+#ifndef __SPARSE_H__
```

Comment:
Does this header need a copyright ?

Reply:
I don't know what it's supposed to be.

Reply:
The current file is based on original header from AOSP - https://android.googlesource.com/kernel/lk/+/5c761131621480d1288692a554805c8cc3fb5442%5E!/
Looks the same in the latest version - https://android.googlesource.com/platform/system/core/+/master/libsparse/sparse_format.h